__tests__/app.test.js

```diff
@@ -40,6 +40,48 @@ describe('GET /api', () => {
     });
 });
 
+describe('GET /api/articles/:article_id', () => {
+    it('200: responds with article object with correct properties', () => {
+        const testId = 1
```

Comment:
This is not a dynamic test, there is no need for this to be declared as a variable.